docs/content/mongocxx-v3/installation/_index.md

```diff
@@ -36,6 +36,66 @@ The Mongo C++ driver is available in the following package managers.
 - [Vcpkg](https://vcpkg.io/) (search for mongo-cxx-driver)
 - [Conan](https://conan.io/center/recipes/mongo-cxx-driver)
 
+### Vcpkg Install Instructions
+
+If you do not already have Vcpkg installed, install it with the following
+command:
+```
+$ git clone https://github.com/Microsoft/vcpkg.git
+$ cd vcpkg
+$ ./bootstrap-vcpkg.sh
+```
+
+Optionally, to install with Visual Studio integration:
+```
+vcpkg integrate install
+```
+
+Install the driver.
+```
+$ ./vcpkg install mongo-cxx-driver
+```
+
+You can find the header files in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/include/
+```
+
+The library files are in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/lib/
+```
+
+### Conan Install Instructions
+
+Package Specifier: `mongo-cxx-driver/3.8.0`
```

Comment:
3.8.1 is now available (as of yesterday)

Reply:
I currently have a PR open with Conan to update their version to 3.8.1. I'll update the version in the instructions once it gets merged: https://github.com/conan-io/conan-center-index/pull/20504